Lua/Lua.sublime-syntax

```diff
@@ -220,8 +210,15 @@ contexts:
 
   if-block:
     - meta_scope: meta.block.lua
-    - match: (?=(?:end|elseif|else){{identifier_break}})
-      pop: true
+    - include: end
+    - match: elseif{{identifier_break}}
+      scope: keyword.control.conditional.lua
+      set:
+        - if-then
+        - expression
+    - match: else{{identifier_break}}
+      scope: keyword.control.conditional.lua
+      set: if-block
```

Comment:
This does ... nothing? Besides stacking `meta.block.lua`, I guess.

Reply:
It probably isn't really important, but this will end up allowing multiple `else` clauses, right?

Reply:
Yes, and/or `elseif` after `else`. If we care, we could add another context to mark `else`/`elseif` after `else` as errors.

Reply:
Eh, the more we get into trying to write a half-baked parser, the more complex and brittle it will get. I think it is probably fine as of now.